week-1-practise/level-3/1-piping.js

```diff
@@ -11,6 +11,7 @@
 */
 
 var startingValue = 5;
+ result = (format( add (multiply (5,2),10)));
```

Comment:
We don't normally leave spaces between the function name and the parameter list

```suggestion
 result = format( add( multiply(5,2), 10));
```